packages/apputils-extension/src/shortcuts.tsx

```diff
@@ -62,19 +62,22 @@ export function displayShortcuts(options: IOptions) {
 
   function formatKeys(keys: string[]): JSX.Element {
     const topContainer: JSX.Element[] = [];
-    for (const key of keys) {
+    keys.forEach((key, index) => {
       const container: JSX.Element[] = [];
       for (const ch of key.split(' ')) {
         container.push(
-          <span className={SHORTCUT_KEY_CLASS}>
+          <span className={SHORTCUT_KEY_CLASS} key={ch}>
             <kbd>{ch}</kbd>
           </span>,
-          <> + </>
+          <React.Fragment key={`${ch}-fragment`}> + </React.Fragment>
         );
       }
       /*topContainer.push(<span>{container.slice(0, -1)}</span>, <>,</>);*/
```

Comment:
I know it was not added in this PR, but let's remove the old comment too (as if it was to be un-commented in the future, it would have the same issue).

```suggestion
```